config/database.yml

```diff
@@ -3,16 +3,20 @@ default: &default
   encoding: unicode
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   host: localhost
-  username: <%= ENV['RAILS_DATABASE_USER'] %>
-  password: <%= ENV['RAILS_DATABASE_PASSWORD'] %>
+  username: postgres
+  password: "24434"
```

Comment:
- Since all of us have different usernames for our PostgreSQL database. It would be better if you make a `.env` file at root directory where you can save your postgresql connection parameters like `username` and `password`. For example:
Open `.env` with your editor and paste these lines
```sh
RAILS_DATABASE_USER=<Put your postgresql username here>
RAILS_DATABASE_PASSWORD=<Put your password here>
JWT_SECRET_KEY=<Put your JWT secret key here>
```

- Now, you can use those parameters in `database.yml` by specifying the environment variable keys like the following:
```ruby
  username: <%= ENV['RAILS_DATABASE_USER'] %>
  password: <%= ENV['RAILS_DATABASE_PASSWORD'] %>
```